### ISSUE TITLE
Add support for more colors

### ISSUE THREAD

**Issue by bhavyaagg** [coslos/LiveWhiteboard-Website#7]
New colors can be added for the user to draw with.

**Comment by JesseKuntz** [comment]
@bhavyaagg I was thinking a color wheel would be a good thing to add (maybe with hex functionality). Does that sound along the lines of what you were thinking?

**Comment by bhavyaagg** [comment]
@JesseKuntz Yes i think that would be great. We will think about adding the same functionality in the app as well. Whatsay @apoorvaagupta ?

**Comment by bhavyaagg** [comment]
@JesseKuntz You can start working on this.

**Comment by buddhikac96** [comment]
i think its better adding a color pallet.. Shall i take this..?

**Comment by bhavyaagg** [comment]
Hey @buddhikac96 Thank you for showing interest. Color pallet would also work just fine. Please allow a few hours to @JesseKuntz if he wants to complete this. You can start in a few hours if he does not reply.

**Comment by bhavyaagg** [comment]
Hey @buddhikac96 . I am sorry to inform you but i think @JesseKuntz has already completed that feature. You can check his PR. I think by increasing the pointer thickness, the coordinates of the points are getting distanced from each other because of which the line would be becoming less clear.

**Comment by bhavyaagg** [comment]
Hey @buddhikac96 I think you can start with the color issue if you like.

**Comment by buddhikac96** [comment]
It's okay.. anyway thank you a lot for the help...  shall i take that..?

**Comment by JesseKuntz** [comment]
@buddhikac96 Feel free to take this issue (the color pallet), as I am still working on finishing up the line thickness.

**Comment by buddhikac96** [comment]
okay guys.... @bhavyaagg  @JesseKuntz

**Comment by bhavyaagg** [comment]
@buddhikac96 Go ahead. All the best :)

**Comment by apoorvaagupta** [comment]
@buddhikac96 Go ahead. You can add the color palette.
